go-controller/pkg/config/config.go

```diff
@@ -988,7 +988,7 @@ var OVNK8sFeatureFlags = []cli.Flag{
 		Name:        "egressip-reachability-total-timeout",
 		Usage:       "EgressIP node reachability total timeout in seconds (default: 1)",
 		Destination: &cliConfig.OVNKubernetesFeature.EgressIPReachabiltyTotalTimeout,
-		Value:       1,
+		Value:       10,
```

Comment:
so if this was defaulted to 1 previously, it would take 7-12 seconds for the timeout to actually happen? The documentation in the code says:
"EgressIP node reachability total timeout in seconds (default: 1)"

Do we have documentation somewhere else that tells the user it will be 7 to 12x this value? If so, do we need to update documentation to indicate the definition of this timeout has changed (upstream and/or downstream)?

Reply:
Well in my opinion the definition of the documentation was wrong but becomes accurate with this PR in way that we don't need to update it. But I guess we can try to improve if you think it is not clear enough.

Reply:
I think the source of many of user headaches is that this was not behaving as documented.

Reply:
OK I cant find any documentation in openshift describing it. Looks like we are OK to just clarify it here.